### PR TITLE
Minor fixes in stores

### PR DESCRIPTION
**Pull request by netopyr**
Fixed some minor issues that I noticed while working on other code.

### CODE REVIEW

**Review by iwsimon — DISMISSED** [review]
LGTM

File: hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/dispatcher/ReadableStoreFactory.java [hunk 1]
```diff
@@ -58,6 +60,8 @@ public class ReadableStoreFactory {
     // e.g. services have to register their stores
     private static final Map<Class<?>, StoreEntry> STORE_FACTORY = Map.of(
             ReadableAccountStore.class, new StoreEntry(TokenService.NAME, ReadableAccountStoreImpl::new),
+            ReadableNftStore.class, new StoreEntry(TokenService.NAME, ReadableAccountStoreImpl::new),
+            ReadableStakingInfoStore.class, new StoreEntry(TokenService.NAME, ReadableAccountStoreImpl::new),
```
Comment (rbair23) [inline hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/dispatcher/ReadableStoreFactory.java:64]: I assume `ReadableAccountStoreImpl` implements the `ReadableNftStore` and `ReadableStakingInfoStore` interfaces

**Review by rbair23 — APPROVED** [review]

File: hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/dispatcher/ReadableStoreFactory.java (hunk 1 above)
Comment (netopyr) [reply hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/dispatcher/ReadableStoreFactory.java]: Good catch! This was probably a copy-and-paste error. Fixed it and added tests.

**Review by povolev15 — DISMISSED** [review]
LGTM

**Review by povolev15 — APPROVED** [review]
LGTM

**Review by rbair23 — APPROVED** [review]

**Pull request merged**